package.json

```diff
@@ -92,65 +93,72 @@
     "redux-thunk": "^2.2.0",
     "reselect": "2.5.4",
     "serialize-javascript": "^1.4.0",
-    "slug": "^0.9.1",
     "source-map-support": "^0.5.8",
     "validator": "^9.2.0",
     "victory": "^0.25.7",
     "websocket.js": "^0.1.7"
   },
   "devDependencies": {
+    "@babel/core": "^7.3.3",
+    "@babel/plugin-proposal-class-properties": "^7.3.3",
+    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
+    "@babel/preset-env": "^7.3.1",
+    "@babel/preset-flow": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-stage-2": "^7.0.0",
     "assets-webpack-plugin": "^3.5.1",
-    "babel-core": "^6.23.1",
-    "babel-eslint": "^8.0.1",
-    "babel-jest": "^21.0.2",
-    "babel-loader": "^7.1.1",
+    "babel-eslint": "^10.0.1",
+    "babel-jest": "^24.1.0",
+    "babel-loader": "^8.0.5",
     "babel-plugin-lodash": "^3.2.11",
     "babel-plugin-transform-import-to-require": "^1.0.0",
     "babel-polyfill": "^6.23.0",
     "babel-preset-latest": "^6.22.0",
-    "babel-preset-react": "^6.23.0",
+    "babel-preset-react": "^6.24.1",
     "babel-preset-stage-2": "^6.22.0",
```

Comment:
With the new `.babelrc` I think these three can be tossed?
```
    "babel-preset-latest": "^6.22.0",
    "babel-preset-react": "^6.24.1",
    "babel-preset-stage-2": "^6.22.0"
```

All the new presets are `@babel/` presets for flow, env and react, so if these 3 presets aren't used anywhere else I think they serve no purpose.

I also think this plugin is unused, as it's no longer used in the `.babelrc`
```
"babel-plugin-transform-import-to-require": "^1.0.0",
```

Reply:
Yee, I guess so :ok_hand: